crates/turbopack/tests/node-file-trace/integration/mdx/index.js

```diff
@@ -0,0 +1 @@
+import Example from "./example.mdx";
```

Comment:
Doesn't this break in node.js?

Also this test case is not listed in the `node-file-trace.rs` test file which calls all test cases.

Reply:
It's existed for testing directly via `cargo run --bin node-file-trace`. I didn't have any idea how to test it with node-file-trace. Do you have any suggestions? Should I remove these files?

Reply:
It should be added here
https://github.com/vercel/turbo/blob/7baa4d41f3efb5eb13ed48eec2ea8911e0759ab2/crates/turbopack/tests/node-file-trace.rs#L46-L240

Reply:
@ForsakenHarmony I know it could be added here, but it could not be executed by Node.js directly, so the test case would always fail

Reply:
That's not a good idea imo, maybe we should have snapshot tests for mdx